src/main/java/plugily/projects/buildbattle/arena/states/build/InGameState.java

```diff
@@ -254,8 +239,9 @@ private void handleThemeVoting(BuildArena pluginArena) {
   }
 
   private void handleBuildTime(BuildArena pluginArena) {
+    int timer = pluginArena.getTimer();
```

Comment:
timer is only used once

Reply:
`pluginArena#getTimer` method invokes `Map#get` which again pressures the CPU performance.

Reply:
also did not saw the loop here, sorry